xee/ext.py

```diff
@@ -555,11 +559,19 @@ def get_variables(self) -> utils.Frozen[str, xarray.Variable]:
     lon_grid = self.project((0, 0, v0.shape[1], 1))
```

Comment:
Hey @boothmanrylan we figured out that the issue is with `pixelLonLat()` actually as it gives the coordinates always into the `degree` so instead of that if you can use this `lnglat_img = ee.Image.pixelCoordinates(ee.Projection(self.crs_arg))` then it gives the coordinates into `given crs` format. 

Also change the bandIds from `longitude` to `x`  into at line [558](https://github.com/google/Xee/pull/97/files#diff-b442e5493bb6c903c13f3d0a2cdab640ad76768c7f413462c89465a11a483c25L558).
-> Same with `latitude` to `y`.

Reply:
Done.